<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Track</class>
 <widget class="QDialog" name="Track">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Phone:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;City:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Address:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;State:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Surface:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Track Name:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Length:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="trackState">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AZ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ME</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ND</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NJ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WY</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="trackName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="trackLength">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="trackAddress">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="trackCity">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Zip:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="trackZip">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="trackPhone">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="trackSurface">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/Resources/Images/Resources/Images/AddTrack2.png</pixmap>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label_7</zorder>
  <zorder>label_6</zorder>
  <zorder>label_8</zorder>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>trackName</zorder>
  <zorder>trackLength</zorder>
  <zorder>trackAddress</zorder>
  <zorder>trackCity</zorder>
  <zorder>label_10</zorder>
  <zorder>trackState</zorder>
  <zorder>trackZip</zorder>
  <zorder>trackPhone</zorder>
  <zorder>trackSurface</zorder>
  <zorder>saveButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_5</zorder>
 </widget>
 <tabstops>
  <tabstop>trackName</tabstop>
  <tabstop>trackLength</tabstop>
  <tabstop>trackSurface</tabstop>
  <tabstop>trackAddress</tabstop>
  <tabstop>trackCity</tabstop>
  <tabstop>trackState</tabstop>
  <tabstop>trackZip</tabstop>
  <tabstop>trackPhone</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
